src/main/java/github/scarsz/discordsrv/modules/voice/VoiceModule.java

```diff
@@ -177,14 +177,14 @@ private void tick() {
                     continue;
                 }
 
-                if (member.getVoiceState() == null || member.getVoiceState().getChannel() == null) {
+                if (member.getVoiceState() == null || member.getVoiceState().getChannel() == null || !(member.getVoiceState().getChannel() instanceof VoiceChannel)) {
```

Comment:
I'm wondering if it would be possible to support Stage channels with voice (for the voice lobby) - there is probably some common interface that has the appropriate methods

Reply:
Stage channels cannot be contained in a category, due to discord limitations, so the entire logic here would need a rework, correct? feels like this would be better as a follow-up PR/feature since it would change how the library works.

Reply:
Stage channels can be in a category......

Reply:
hmm, maybe I was mistaken! I thought I remembered seeing that restriction in JDAs docs, but I must have misinterpreted it

Reply:
The voice module shouldn't require the lobby to be in the category either way (although the documentation regarding that may be misleading)

Reply:
Out of scope of this PR so I'm resolving this conversation.